source/documentation/deploying_services/use_a_custom_domain.md

```diff
@@ -1,10 +1,12 @@
-# Set up a custom domain using the cdn-route service
+# Managing custom domains using the cdn-route service
```

Comment:
There are links that references this title by anchor elsewhere in the docs. They need updating (`deploying_services/configure_cdn.md` and `deploying_apps/names_routes_domains.md`). Having said that, those 2 links don't need the anchor at all as this is the top of the page, so it could be omitted.

Reply:
changed